RDKB-61612 : WAN Failover Utility is required in Ethwan mode similar to DOCSIS

suriya-prem opened the pull request:
RDKB-61612 : WAN Failover Utility is required in Ethwan mode similar to DOCSIS

Reason for change: Enable WAN Failover simulation in EthWAN mode
Test Procedure: Set this "Device.X_RDKCENTRAL-COM_EthernetWAN.LinkDown" TR181 param to true for verifying the WAN failover simulation is happening or not when device is in EthWAN mode.
Risks: Low
Priority: P1
Signed-off-by: Suriyanarayanan_MP@comcast.com

Commit:
RDKB-61612 : WAN Failover Utility is required in Ethwan mode similar to DOCSIS
Reason for change: Enable WAN Failover simulation in EthWAN mode
Test Procedure: Set this "Device.X_RDKCENTRAL-COM_EthernetWAN.LinkDown" TR181 param to true for verifying the WAN failover simulation is happening or not when device is in EthWAN mode.
Risks: Low
Priority: P1
Signed-off-by: Suriyanarayanan_MP@comcast.com

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/utopia?pullRequest=97) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/utopia?pullRequest=97) before we can accept your contribution.<br/><hr/>**smp960** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/utopia?pullRequest=97) it.</sub>

suriya-prem committed:
RDKB-61612 : WAN Failover Utility is required in Ethwan mode similar to DOCSIS
    Reason for change: Enable WAN Failover simulation in EthWAN mode
    Test Procedure: Set this "Device.X_RDKCENTRAL-COM_EthernetWAN.LinkDown" TR181 param to true for verifying the WAN failover simulation is happening or not when device is in EthWAN mode.
    Risks: Low
    Priority: P1

aadhithan01 committed:
Merge branch 'develop' into feature/wan_failover_sim

snayak002c committed:
Merge branch 'develop' into feature/wan_failover_sim

snayak002c committed:
Merge branch 'develop' into feature/wan_failover_sim
